tests/test_methods_and_attributes.py

```diff
@@ -431,6 +431,17 @@ def test_accepts_none(msg):
     assert "incompatible function arguments" in str(excinfo.value)
 
 
+def test_casts_none(msg):
```

Comment:
Post-merge comment: `msg` is unused and can be removed. Looks like a copy-paste accident.
(I'll create a PR to fix.)